cloverage/src/cloverage/coverage.clj

```diff
@@ -98,6 +99,8 @@
         "Generate a JSON report for Codecov.io" :default false]
        ["--[no-]coveralls"
         "Send a JSON report to Coveralls if on a CI server" :default false]
+       ["--[no-]junit"
+        "Output test results as junit xml file" :default false]
```

Comment:
You might want to document this option works only for the `clojure.test` runner.
It would be nicer if users can see warnings or errors by specifying an incompatible combination of runners and reporters. While we can introduce such a generic mechanism, I think ad-hoc handling is enough for now.